src/lib/OpenEXR/ImfStandardAttributes.cpp

```diff
@@ -65,6 +65,10 @@ OPENEXR_IMF_INTERNAL_NAMESPACE_SOURCE_ENTER
 IMF_STD_ATTRIBUTE_IMP (originalDataWindow, OriginalDataWindow, Box2i)
 IMF_STD_ATTRIBUTE_IMP (worldToCamera, WorldToCamera, M44f)
 IMF_STD_ATTRIBUTE_IMP (worldToNDC, WorldToNDC, M44f)
+IMF_STD_ATTRIBUTE_IMP (sensorCenterOffset, SensorCenterOffset, V2f)
+IMF_STD_ATTRIBUTE_IMP (sensorOverallDimensions, SensorOverallDimensions, V2f)
+IMF_STD_ATTRIBUTE_IMP (sensorPhotositePitch, SensorPhotositePitch, float)
```

Comment:
Although it is most common, perhaps there are cases where the pitch is not identical horizontally and vertically? Or there is even perhaps no simple horizontal/vertical definition as in the case of slanted/irregular grid arrays like [Fujifilm Super CCD](https://en.wikipedia.org/wiki/Super_CCD) (and more modern small/large pixel HDR equivalents) or [Fujifilm EXR](https://en.wikipedia.org/wiki/Bayer_filter#Fujifilm_%22EXR%22_color_filter_array)?

Reply:
I can add a note when I go back in to add content on the .rst page for CCT and tint and all that.

In all the SMPTE RIS discussions on metadata for camera tracking, no one has ever asked to support, e.g., the sensor on the Sony F65, which is at a 45º angle. And the last time I saw non-square pixels in acquisition was pre-Rec. 709 HDTV cameras (the Wikipedia article for "Grand Alliance (HDTV)" supports this).

While I understand that OpenEXR use is not limited to the entertainment industry, AFAIK it is by far the dominant use case, and we should not let the perfect be the enemy of the good.

Reply:
> While I understand that OpenEXR use is not limited to the entertainment industry, AFAIK it is by far the dominant use case, and we should not let the perfect be the enemy of the good.

Of course. Listing the known assumptions/"limitations" is part of this good practice.

Reply:
If it's really needed a sensorPhotositeAspectRatio can be added later, I suppose.

Perhaps more relevant is clarifying what to do about anamorphic lenses. In that case photosites don't have the same field of view horizontally and vertically due to the lens optics, rather than their dimensions. pixelAspectRatio defines how the image should be displayed, not what the camera lens actually did, so is generally only indicative of how much anamorphic squeeze happened at capture time. It seems like all the lens attributes assume non-anamorphic lenses